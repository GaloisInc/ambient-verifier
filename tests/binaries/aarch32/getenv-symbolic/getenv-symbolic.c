#include <stdlib.h>
#include "ambient_assert.h"

int main(void) {
  char *symbolic_val = getenv("SYMBOLIC");

  for (int i = 0; i < 2; i++) {
    char symbolic_char = symbolic_val[i];
    ambient_assert(symbolic_char || !(symbolic_char));
    ambient_assert(symbolic_char == symbolic_char);
  }
  ambient_assert(symbolic_val[2] == '\0');

  return 0;
}
