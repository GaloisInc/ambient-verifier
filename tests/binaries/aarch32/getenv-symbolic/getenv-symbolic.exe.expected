successful: 0
failed: 0
overrideDir: tests/binaries/aarch32/getenv-symbolic/overrides
